### ISSUE TITLE
TEdit Round 5:  TEDIT.FORMAT.HARDCOPY fails on medley/docs/internal/StackFig.sketch

### ISSUE THREAD

**Issue by fghalasz** [Interlisp/medley#1905]
```
(SETQ S (OPENTEXTSTREAM 'medley/docs/internal/StackFig.sketch))
(TEDIT.FORMAT.HARDCOPY S 'TEST T NIL NIL NIL 'PDF)
```

This "fails".  It doesn't break, it just prints "Formatting for print..." and then hangs forever - never to return.

Works just fine on master.

Discovered as part of HCFILES failure to run to completion on TEdit Round 5.

**Comment by rmkaplan** [comment]
Rebuild and try it again

**Comment by fghalasz** [comment]
OK latest push fixes medley/docs/internal/StackFig.sketch issue.

do_hcfiles.sh script now runs to completion on TEdit Round 5.  I have not compared "fails" between master and tedit-round-5.

Fixed with commit d29f6c489101e9593fadc3c59319a9dbd8e4a6b8.
